src/browser/ui/dom/DOMProperty.js

```diff
@@ -70,6 +70,8 @@ var DOMPropertyInjection = {
       );
     }
 
+    DOMProperty.getAttributeNamespace = domPropertyConfig.PropertyNamespaces;
```

Comment:
This overrides getAttributeNamespace each time injectDOMPropertyConfig is called, which means the HTML and SVG configs won't be merged properly. If you assign this in the loop below like the other lookup tables do then it should do the right thing.